src/benchmarklib/cli_config_parser.cpp

```diff
@@ -88,9 +88,14 @@ BenchmarkConfig CLIConfigParser::parse_cli_options(const cxxopts::ParseResult& p
     std::cout << "- Encoding is '" << encoding_type_str << "'" << std::endl;
   }
 
-  const auto indexes = parse_result["indexes"].as<bool>();
-  if (indexes) {
-    std::cout << "- Creating indexes (as defined by the benchmark)" << std::endl;
+  const auto chunk_indexes = parse_result["chunk_indexes"].as<bool>();
+  if (chunk_indexes) {
+    std::cout << "- Creating chunk indexes (separate index per chunk; columns defined by benchmark)" << std::endl;
+  }
+
+  const auto table_indexes = parse_result["table_indexes"].as<bool>();
+  if (table_indexes) {
```

Comment:
Should we check here that not both indexes are requested or is this fine?

Reply:
I'm not sure if this will be an issue.
But when chunk indexes were removed it won't matter anymore.

Reply:
I added a warning.